WcaOnRails/app/assets/stylesheets/persons.scss

```diff
@@ -0,0 +1,34 @@
+#persons-query-fields {
+  #search-box {
+    margin: 15px 0;
+  }
+}
+
+.persons-table {
+  .name {
+    width: 40%;
+  }
+
+  .wca-id {
+    width: 10%;
+  }
+
+  .country {
+    width: 30%;
+  }
+
+  .competitions-count,
+  .podiums-count {
+    width: 10%;
+    text-align: center;
+  }
+}
+
+.fixed-table-loading,
+.pagination-detail {
+  display: none !important;
```

Comment:
For the record, I guessed the syntax from reading https://github.com/brigade/scss-lint#disabling-linters-via-source, but I did test that putting the disable comment at the end of a line seems to work.